docs/demos/coast-to-coast-q4-2023.md

```diff
@@ -0,0 +1,493 @@
+  mkdir -p manifests
+  kustomize build config/default > manifests/manifests.yaml
+  docker build -t "quay.io/operator-framework/coastal-bundle:${version}" -f plainbundle.Dockerfile .
+  docker push "quay.io/operator-framework/coastal:${version}"
+  docker push "quay.io/operator-framework/coastal-bundle:${version}"
+done
+```
+
+### Create an initial File-Based Catalog (FBC) Image
+Now we need to build and push an initial File-Based Catalog (FBC) image containing
+our bundles. To help highlight the new polling functionality, we will generate a catalog containing
+only the `v1.0.0-alpha1` bundle and we will mimic active development by incrementally updating the catalog
+image to include more of our bundles.
+
+- Create a `catalog/` directory
+```sh
+mkdir catalog
+```
+
+- Create a Dockerfile for the catalog image
+```sh
+cat << EOF > catalog.Dockerfile
+FROM scratch
+ADD catalog /configs
+LABEL operators.operatorframework.io.index.configs.v1=/configs
+EOF
+```
+
+- Create the FBC YAML file
+```sh
+cat << EOF > catalog/index.yaml
+schema: olm.package
+name: coastal
+---
+schema: olm.bundle
+name: coastal.v1.0.0-alpha1
+package: coastal
+image: quay.io/operator-framework/coastal-bundle:v1.0.0-alpha1
+properties:
+  - type: olm.package
+    value:
+      packageName: coastal
+      version: 1.0.0-alpha1
+  - type: olm.bundle.mediatype
+    value: plain+v0
+---
+schema: olm.channel
+name: stable
+package: coastal
+entries:
+  - name: coastal.v1.0.0-alpha1
+EOF
+```
+
+- Build and push the catalog image
+```sh
+docker build -t quay.io/operator-framework/coastal-catalog:latest -f catalog.Dockerfile . && \
+docker push quay.io/operator-framework/coastal-catalog:latest
+```
+
+### Create a `Catalog` Resource
+Create a `Catalog` resource that references the catalog image we built
+in the previous step and specify a polling interval of 15 seconds. This will make sure that the updates
+we make to the catalog image are reflected on our cluster relatively quickly.
+
+- Create the `Catalog` resource:
+```sh
+kubectl apply -f - <<EOF
+apiVersion: catalogd.operatorframework.io/v1alpha1
+kind: Catalog
+metadata:
+  name: coastal
+spec:
+  source:
+    type: image
+    image:
+      ref: quay.io/operator-framework/coastal-catalog:latest
+      pollInterval: 15s
+EOF
+```
+
+## Only install/upgrade to `v1.0.z` releases
+In this section we are going to create an `Operator` resource that attempts to install our Operator
+with a version range of `1.0.x`. This version range ensures we only install z-stream releases for `v1.0`
+excluding pre-release versions.
+
+- Create the `Operator` resource:
+```sh
+kubectl apply -f - <<EOF
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  name: coastal
+spec:
+  packageName: coastal
+  version: 1.0.x
+EOF
+```
+
+We should see the `Operator` resource eventually has a failed resolution status. Verify this with:
+```sh
+kubectl get operator/coastal -o yaml -w
```

Comment:
Can we please also post the sample payload returned, so that it can be reviewed offline, without running the command to see what's inside the `.status`?

This applies to all the `kubectl get` commands in this doc.

Reply:
Should be done as of https://github.com/operator-framework/operator-controller/pull/533/commits/3acecb5e0c788b16afdc7e5de5e307798b390691